weed/server/volume_server_handlers_write.go

```diff
@@ -87,6 +89,13 @@ func (vs *VolumeServer) DeleteHandler(w http.ResponseWriter, r *http.Request) {
 		count = chunkManifest.Size
 	}
 
+	n.LastModified = uint64(time.Now().Unix())
+	if len(r.FormValue("time")) > 0 {
```

Comment:
"time" is too generic. we can use "LastModified" here to be clear about the meaning and the unit.